src/client/pfs/pfs.proto

```diff
@@ -352,4 +360,19 @@ service BlockAPI {
   rpc ListBlock(ListBlockRequest) returns (BlockInfos) {}
 }
 
+message PutObjectRequest {
+  bytes value = 1;
+  repeated Tag tags = 2;
+}
 
+service ObjectAPI {
+  rpc PutObject(stream PutObjectRequest) returns (Object) {}
+  rpc GetObject(Object) returns (google.protobuf.BytesValue) {}
+  rpc GetTag(Tag) returns (google.protobuf.BytesValue) {}
+  rpc Compact(google.protobuf.Empty) returns (google.protobuf.Empty) {}
+}
```

Comment:
Actually, it occurred to me that we also need an `Exists` API that simply checks for the existence of a tag.  This is useful for a job to check if a datum needs to be reprocessed.